utils/CI/fix_whitespace.sh

```diff
@@ -7,7 +7,9 @@ make -C data/tools reindent
 if [[ "$OS" == "Linux" ]]; then
   find src/ -name \*.\[ch\]pp -print0 | xargs -0 sed -i 's/[[:blank:]]*$//'
   find data/ -name \*.lua -print0 | xargs -0 sed -i 's/[[:blank:]]*$//'
+  find data/ -name \*.lua -print0 | xargs -0 data/tools/check_mixed_indent
 else
   find src/ -name \*.\[ch\]pp -print0 | xargs -0 sed -i '' 's/[[:blank:]]*$//'
   find data/ -name \*.lua -print0 | xargs -0 sed -i '' 's/[[:blank:]]*$//'
+  find data/ -name \*.lua -print0 | xargs -0 data/tools/check_mixed_indent
```

Comment:
This can be present once outside of the if/else I think? The if/else exists because there's a difference in the `sed` syntax between Linux and macOS, but this is a python script.